spec/features/user_signs_up_spec.rb

```diff
@@ -63,15 +61,18 @@
     scenario 'failure because email in use' do
       create_email_address_for(create_user('user'), "bob@bob.edu")
       visit signup_path
-      complete_signup_email_screen("Instructor","bob@bob.edu")
-      expect(page).to have_content 'email_in_use'  # TODO fix when make proper alert
+      select 'Instructor', from: "signup_role"
+      fill_in (t :"signup.start.email"), with: "bob@bob.edu"
+      click_button(t :"signup.start.next")
+      expect(page).to have_content 'Email already in use'
     end
 
     scenario 'failure because suspected non-school email then success' do
       visit signup_path
-      complete_signup_email_screen("Instructor","bob@gmail.com")
-      skip # TODO Nathan should this be a JS check to see that email is not @blah.edu?
-      # checking in BE might be a waste
+      select 'Instructor', from: "signup_role"
+      fill_in (t :"signup.start.email"), with: "bob@gmail.com"
+      click_button(t :"signup.start.next")
```

Comment:
yeah, it's happening on the first click.  then the warning message is displayed if the email isn't edu.  I updated `complete_signup_email_screen` to do two clicks, but that means that it can't be used to actually test the message so this spec needs to do the steps itself.